src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    
+    .nav__item {
+      position: relative;
+      overflow: hidden;
+    }
+    
+    .nav__link {
+      display: block;
+      text-decoration: none;
+      text-transform: uppercase;
+      font-weight: 500;
+      padding: 23px 10px;
```

Comment:
```suggestion
      font-weight: 500;
      padding: 23px 0;
      position relative
```
for example this comment and comment above on .nav__item can help to fix these tests
![Screenshot from 2023-04-07 11-50-04](https://user-images.githubusercontent.com/33029864/230577646-3b66acd6-6a44-4094-95fc-0e04144861b5.png)